### PR TITLE
fixed menuloop and created new Databaseconnection Class

### PR DESCRIPTION
**Pull request by flip1012**
I created a new class for the initializing the database connection and reorganized the menuLoop() so that the login is only called once and works with all the table changes we want to make.

I have also made it so that the Loop will now ask if you want to keep working on the current database, with a "y" for yes or "n" for no. and then it will ask if you want to work on another database other than your local one.

I was not able to incorporate the the change where the database will use the user logins to provide restricted access to the tables they are not supposed to access so I will need more time for this to work, hopefully it will be ready in the future.

### CODE REVIEW

**Review by jfelici4 — APPROVED** [review]
Works great. The merging will be tricky with #61 @mesta88 do you want me to figure out how to merge this after you finish reviewing and merging #61? (And after you finish reviewing this too of course.)

**Review by mesta88 — APPROVED** [review]
Really like the new menu structure. So far it seems much more seamless and user-friendly, especially with a single log-in. Nice work!

**Pull request merged**
